include/geometry.tcc

```diff
@@ -11,15 +11,15 @@ inline Eigen::Matrix<double, 2, 2> mpm::Geometry<2>::rotation_matrix(
 
   // clang-format off
   const Eigen::Matrix<double, 2, 2> rotation_matrix = 
-    (Eigen::Matrix<double, 2, 2>() << cos_alpha_cos_beta - sin_alpha_sin_beta,  -cos_alpha_sin_beta - cos_alpha_cos_beta,
+    (Eigen::Matrix<double, 2, 2>() << cos_alpha_cos_beta - sin_alpha_sin_beta,  -cos_alpha_sin_beta - sin_alpha_cos_beta,
```

Comment:
Could you add a unit test which is not 45*?

Reply:
Done :)